Decred Wallet/Utils/LocalizedStrings.swift

```diff
@@ -142,16 +133,24 @@ struct LocalizedStrings {
     static let total = NSLocalizedString("total", comment: "")
     static let stageTimes = NSLocalizedString("stageTimes", comment: "")
     static let tapToViewInformation = NSLocalizedString("tapToViewInformation", comment: "")
+    static let walletStatus = NSLocalizedString("walletStatus", comment: "")
+    static let walletSynced = NSLocalizedString("walletSynced", comment: "")
+    static let walletNotSynced = NSLocalizedString("walletNotSynced", comment: "")
+    static let syncStageDescription = NSLocalizedString("syncStageDescription", comment: "")
+    
+    
```

Comment:
It's customary to use 1 empty line space for separation of concerns. There are 2 here.

Reply:
resolved in 2766f8f